### ISSUE TITLE
Article resnet34 model training batch size

### ISSUE THREAD

**Issue by guissart** [joojs/fairface#10]
Hello,
In the Arxiv article you didn't mention the batch size during the training.

> To compare model performance of different datasets, weused an identical model architecture, ResNet-34 [18], to betrained from each dataset.   We used ADAM optimization[29] with a learning rate of 0.0001. Given an image, we de-tected faces using the dlib1’s CNN-based face detector [28]and ran the attribute classifier on each face. The experimentwas done in pyTorch.

With what batch size did you get the of the paper ?

Thank you
